### ISSUE TITLE
Internal error in xmldoc when hovering over Fulma's Box.box'

### ISSUE THREAD

**Issue by kerams** [dotnet/fsharp#11598]
**Repro steps**

Hover over [`Box.box'`](https://github.com/Fulma/Fulma/blob/master/src/Fulma/Elements/Box.fs) to try to see its xmldoc.

**Expected behavior**

Tooltip with `Generate <div class="box"></div>`.

**Actual behavior**

![devenv_MmHAZY97i1](https://user-images.githubusercontent.com/5063478/119252635-a6605700-bbad-11eb-9e00-8804b456d528.png)

**Related information**

* Win 10
* .NET 5.0.300-preview.21258.4
* VS 2019 16.10 Preview 4.0

**Comment by cartermp** [comment]
Yeah, the `'` is disallowed by the xml reader used to process the comments. Fun!
